htlcswitch/link.go

```diff
@@ -1464,6 +1481,14 @@ func (l *channelLink) htlcManager() {
 						" %v", err),
 				)
 			}
+		case qReq := <-l.newQuiescenceRequests:
+			l.activeQuiescenceRequest = qReq
+			err := l.quiescer.initStfu()
+			if err != nil {
+				close(qReq)
```

Comment:
See comment about potentially going to `fn.Result[bool]` here for the return value to be able to thread thru more error content.